libifbtsvm/libifbtsvm.py

```diff
@@ -65,19 +65,16 @@ def _compute_score(cls, score, c):
 
     @staticmethod
     def _decrement(candidates, score, alphas, fuzzy, data) -> Tuple[np.ndarray, np.ndarray, np.ndarray, np.ndarray]:
-        """
+        candidates_for_alphas = score[0][candidates].astype(int)
+        alphas = np.delete(alphas, candidates_for_alphas)
+        fuzzy = np.delete(fuzzy, candidates_for_alphas)
+        data = np.delete(data, candidates_for_alphas, axis=0)
 
-        :return:
-        """
         sco0 = np.delete(score[0], candidates)
+        for i in range(len(sco0)):  # update index arrary about alphas, after deletion on alphas
```

Comment:
This introduction of a non-vectorized `numpy` function could incur some non-negligible slowdown of the code. 

I haven't dig deep in the `numpy` library's documentation but perhaps there exist some function that would do the trick here. 

I'll let you judge if you think it's outside the scope of the current PR. However, to a minimum, I would request a `# TODO` comment to make sure this is updated eventually.

Reply:
That's a very good point! I didn't think that far. Based on your suggestion, I think we can do vectorize like the new commit, what do you think?

Reply:
Sorry for the delay again (life :sweat_smile:)

The change looks fine to me. :+1: